src/libmeasurement_kit/nettests/runnable.cpp

```diff
@@ -238,74 +238,131 @@ std::string Runnable::generate_output_filepath() {
     return filename.str();
 }
 
+void Runnable::contact_bouncer(Callback<Error> cb) {
+    if (!needs_bouncer) {
+        cb(NoError());
+        return;
+    }
+    auto bouncer = options.get(
+        "bouncer_base_url", std::string{"https://a.collector.ooni.io/bouncer"});
```

Comment:
I would define this string in [include/measurement-kit/ooni/collector_client.hpp](https://github.com/measurement-kit/measurement-kit/blob/master/include/measurement_kit/ooni/collector_client.hpp#L21) instead of hardcoding it here!